src/components/Atoms/Alert/index.stories.jsx

```diff
@@ -0,0 +1,5 @@
+import { Alert } from "./index";
+
+export default { component: Alert };
+
+export const Default = {};
```

Comment:
```suggestion
export const Default = {
　args: {
    　isActive: true,
    　errorText: "text",
 　 },
  };
```
StoryBookに表示させるためには、この記述が必要です。